lib/es5-helpers.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const ChildProcess = require('child_process')
+const Path = require('path')
+const FS = require('fs')
+const find = require('atom-linter').findFile
+
+let prefixPath = null
+
+function findEslintDir(params) {
+  const eslintPathLocal = Path.join(FS.realpathSync(Path.join(__dirname, '..')), 'node_modules', 'eslint')
```

Comment:
Because the value of this doesn't change and syscalls are expensive, can we move this in upper scope please?